pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/ClassTypeResolver.java

```diff
@@ -268,6 +267,7 @@ public Object visit(ASTClassOrInterfaceType node, Object data) {
             }
         }
 
+        // FIXME, we should discard the array depth on this node, it should only be known to ASTReferenceType (#910)
```

Comment:
I'm not sure if this fixme implies this intended for an upcoming PR or left for 7.0.0 for some reason. It's the first one, right?

Reply:
I intended that for an upcoming PR, since I don't think it's really API breaking, somewhat like #949